unit/solvers/smt2_incremental/smt2_incremental_decision_procedure.cpp

```diff
@@ -614,5 +614,36 @@ TEST_CASE(
       smt_declare_function_commandt{index_term, {}},
       smt_assert_commandt{smt_core_theoryt::equal(
         foo_term, smt_array_theoryt::select(array_term, index_term))}};
+    REQUIRE(test.sent_commands == expected_commands);
```

Comment:
I'd suggest putting the fix of the existing test in a separate commit, in order to keep the commit history clean.

Reply:
Separated